src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -176,16 +169,16 @@
         {2, queue_messages_persistent_bytes, gauge, "Size in bytes of persistent messages", queue_messages_bytes_persistent},
         {2, queue_messages_paged_out, gauge, "Messages paged out to disk", queue_messages_paged_out},
         {2, queue_messages_paged_out_bytes, gauge, "Size in bytes of messages paged out to disk", queue_messages_bytes_paged_out},
-        {2, queue_disk_reads, gauge, "Number of times queue read messages from disk", disk_reads},
-        {2, queue_disk_writes, gauge, "Number of times queue wrote messages to disk", disk_writes}
+        {2, queue_disk_reads_total, counter, "Total number of times queue read messages from disk", disk_reads},
+        {2, queue_disk_writes_total, counter, "Total number of times queue wrote messages to disk", disk_writes}
     ]}
 ]).
 
 -define(TOTALS, [
-    {connection_created, connections, gauge, "Total number of connections"},
-    {channel_created, channels, gauge, "Total number of channels"},
-    {consumer_created, consumers, gauge, "Total number of consumers"},
-    {queue_metrics, queues, gauge, "Total number of queues"}
+    {connection_created, connections, gauge, "Connections currently running"},
```

Comment:
This name reads like a counter, I'd go with just `connections`.

Reply:
It is `connections` - it caught me as well earlier.

The ordering differs from the metrics above. The second element in the tuple is the name of the metric, to which `rabbitmq_` gets prepended, so `rabbitmq_connections` is the actual metric name.

Reply:
Maybe a comment would help here?